redis_enterprise/tests/support.py

```diff
@@ -38,6 +38,20 @@
     'rdse.redis_master_sync_in_progress',
     'rdse.redis_mem_fragmentation_ratio',
     'rdse.redis_used_disk',
+    'rdse.bdb_crdt_syncer_ingress_bytes',
```

Comment:
Are you adding metrics to the integration? If so, they will need to be added to the metadata CSV file

Reply:
@emarsha94  - the question is should we add all metrics from all groups? we use ephemeral for testing because not all metrics can be asserted all the time.

Reply:
@j8-redis I am a little confused on this. Are these new metrics being added to the integration? Could you please clarify you previous response? Thanks!

Reply:
@emarsha94  - they are being added. they were present in our metrics.py file, but not in metadata.csv because they are optional. based on your comments they have been added.